tests/acceptance/acceptancesuites/filterboxsuite.js

```diff
@@ -104,7 +104,7 @@ test(`multioption filterbox works with back/forward navigation and page refresh`
 
   // Hit the back button, see the 'Marty' filter disappear
   await browserBackButton();
-  await expectRequestFiltersToEql(filterBoxLogger, {});
+  await expectRequestFiltersToEql(filterBoxLogger, null);
```

Comment:
instead of expecting filters to equal null, can we add a new helper that expects a param to not be in the request?

Reply:
sure

Reply:
that helper is already in there, so I'll use that